Library/Homebrew/utils.rb

```diff
@@ -278,7 +312,7 @@ def puts_columns(items, highlight = [])
   # TTY case: If possible, output using multiple columns.
   console_width = Tty.width
   console_width = 80 if console_width <= 0
-  max_len = items.max_by(&:length).length
+  max_len = items.max_by { |s| Tty.strip_ansi(s).length }.length
```

Comment:
I'd rather not run everything through `Tty.strip_ansi` twice. Why not put

```ruby
plain_item_lengths = items.map { |s| Tty.strip_ansi(s).length }
max_len = plain_item_lengths.max
```

here and …